### PR TITLE
added mixmaster as a build option to the neomutt template

### PR DESCRIPTION
**Pull request by terminaldweller**
#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

### CODE REVIEW

File: srcpkgs/neomutt/template [hunk 1]
```diff
@@ -38,6 +41,13 @@ case "${XBPS_TARGET_MACHINE}" in
 	# Inquiry: Do we have largefile support on everywhere?
 esac
 
+do_configure() {
+	if [ "$build_option_mixmaster" ]; then
+		configure_args+=" --mixmaster"
+	fi
+	./configure ${configure_args}
+}
```
Comment (paper42) [inline srcpkgs/neomutt/template:49]:
```suggestion
if [ "$build_option_mixmaster" ]; then
	configure_args+=" --mixmaster"
fi
```

**Review by paper42 — COMMENTED** [review]
and please adjust the commit message according to CONTRIBUTING.md

**Comment by terminaldweller** [comment]
done

**Comment by paper42** [comment]
and squash the commits, it should be one per package

cc maintainer @Vaelatern

**Comment by Vaelatern** [comment]
I believe this will be enabled by default next release. Do you think that is the case too?

**Comment by terminaldweller** [comment]
you mean the neomutt devs will enable this by default from the next release?

**Comment by terminaldweller** [comment]
> and squash the commits, it should be one per package
> 
> cc maintainer @Vaelatern

It is one commit now. at least it says so at the top. is that acceptable?

**Comment by terminaldweller** [comment]
mixmaster is kinda dead so it doesn't add much to have this enabled by default so I'm closing this. feel free to re-open if anyone disagrees.

**Pull request closed without merging**